app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -143,23 +147,36 @@ export const ScanningQRCodeScreen: React.FC = () => {
           })
           return
         }
-
-        Alert.alert(
-          LL.ScanningQRCodeScreen.invalidTitle(),
-          destination.invalidReason === "InvoiceExpired"
-            ? LL.ScanningQRCodeScreen.expiredContent({
-                found: data.toString(),
-              })
-            : LL.ScanningQRCodeScreen.invalidContent({
+        destination.invalidReason === "InvoiceExpired"
```

Comment:
if the assumption is that, if an invoice is expired then it contains a loadable link, I think this is a wrong assumption.

a destination can be invalid for a couple more reason. I think there should be a specific invalidReason that is something like "isLink" (which would means it's not a proper payment destination), and IIF this is the reason, then we suggest to the user to open the link from the browser

Reply:
If `paymentType` of the `parsedDestination` from QR does not match any, then `invalidReason` defaults to `UnknownDestination`.
https://github.com/GaloyMoney/galoy-mobile/blob/331d9552448bd22038131077cacbc8baddc7d3f6/app/screens/send-bitcoin-screen/payment-destination/index.ts#L63-L68

I can put a check for `UnknownDestination`, but then also QR can contain text other than URL. I can check once again that `data.toString()` is indeed a URL by using this method https://stackoverflow.com/a/43467144

Reply:
Sounds like a good option

Reply:
Committed